Public/Src/Engine/Processes/FileAccessManifest.cs

```diff
@@ -281,21 +459,47 @@ private static void WriteChars(BinaryWriter writer, string str)
             }
         }
 
+        private static string ReadChars(BinaryReader reader)
+        {
+            uint length = reader.ReadUInt32();

```

Comment:
>ReadUInt32 [](start = 33, length = 10)

Since this is used for paths, perhaps it is more effecient to use 7-bit encoded integers.

Reply:
See previous comment about Detours.

---
In reply to: [276813401](https://github.com/Microsoft/BuildXL/pull/159#discussion_r276813401) [](ancestors = 276813401)